exporter/exporterbatcher/sizer_type.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterbatcher // import "go.opentelemetry.io/collector/exporter/exporterbatcher"
+
+import (
+	"fmt"
+)
+
+type SizerType struct {
+	string
+}
+
+const (
+	itemsSizerString = "items"
```

Comment:
Then we can expose these instead on the vars:
```suggestion
	SizerTypeItemst SizerType = "items"
```

Reply:
That's neat. Thanks!